Tests/varLib/instancer_test.py

```diff
@@ -1916,6 +1917,212 @@ def test_normalizeAxisLimits_missing_from_fvar(varfont):
+    # Regular
+    instancer.updateNameTable(varfont, {"wght": 400})
+    names = _get_name_records(varfont)
+    assert names[(1, 3, 1, 0x409)] == "Test Variable Font"
+    assert names[(2, 3, 1, 0x0409)] == "Regular"
+    assert names[(3, 3, 1, 0x0409)] == "2.001;GOOG;TestVariableFont-Regular"
+    assert names[(6, 3, 1, 0x409)] == "TestVariableFont-Regular"
+    assert (16, 3, 1, 0x409) not in names
+    assert (17, 3, 1, 0x409) not in names
+
+    # Black
+    instancer.updateNameTable(varfont, {"wght": 900})
+    names = _get_name_records(varfont)
+    assert names[(1, 3, 1, 0x409)] == "Test Variable Font Black"
+    assert names[(2, 3, 1, 0x409)] == "Regular"
+    assert names[(3, 3, 1, 0x0409)] == "2.001;GOOG;TestVariableFont-Black"
+    assert names[(6, 3, 1, 0x409)] == "TestVariableFont-Black"
+    assert names[(16, 3, 1, 0x409)] == "Test Variable Font"
+    assert names[(17, 3, 1, 0x409)] == "Black"
+
+    # Thin
+    instancer.updateNameTable(varfont, {"wght": 100})
+    names = _get_name_records(varfont)
+    assert names[(1, 3, 1, 0x409)] == "Test Variable Font Thin"
+    assert names[(2, 3, 1, 0x409)] == "Regular"
+    assert names[(3, 3, 1, 0x0409)] == "2.001;GOOG;TestVariableFont-Thin"
+    assert names[(6, 3, 1, 0x409)] == "TestVariableFont-Thin"
+    assert names[(16, 3, 1, 0x409)] == "Test Variable Font"
+    assert names[(17, 3, 1, 0x409)] == "Thin"
+
+    # Thin Condensed
+    instancer.updateNameTable(varfont, {"wdth": 79, "wght": 100})
+    names = _get_name_records(varfont)
+    assert names[(1, 3, 1, 0x409)] == "Test Variable Font Thin Condensed"
+    assert names[(2, 3, 1, 0x409)] == "Regular"
+    assert names[(3, 3, 1, 0x0409)] == "2.001;GOOG;TestVariableFont-ThinCondensed"
+    assert names[(6, 3, 1, 0x409)] == "TestVariableFont-ThinCondensed"
+    assert names[(16, 3, 1, 0x409)] == "Test Variable Font"
+    assert names[(17, 3, 1, 0x409)] == "Thin Condensed"
+
+
+def test_updatetNameTable_axis_order(varfont):
+    axes = [
+        dict(
+            tag="wght",
+            name="Weight",
+            values=[
+                dict(value=400, name='Regular'),
+            ],
+        ),
+        dict(
+            tag="wdth",
+            name="Width",
+            values=[
+                dict(value=75, name="Condensed"),
+            ]
+        )
+    ]
+    buildStatTable(varfont, axes)
+    instancer.updateNameTable(varfont, {"wdth": 75, "wght": 400})
+    names = _get_name_records(varfont)
+    assert names[(17, 3, 1, 0x409)] == "Regular Condensed"
+
+    # Swap the axes so the names get swapped
+    axes[0], axes[1] = axes[1], axes[0]
+
+    buildStatTable(varfont, axes)
+    instancer.updateNameTable(varfont, {"wdth": 75, "wght": 400})
+    names = _get_name_records(varfont)
+    assert names[(17, 3, 1, 0x409)] == "Condensed Regular"
+
+
+def test_updateNameTable_with_multilingual_names(varfont):
+    name = varfont["name"]
+    name.setName("Test Variable Font", 1, 3, 1, 0x405)
```

Comment:
curious what language is 0x405, too lazy to check (a comment maybe)?

Reply:
Czech. I'll add a comment